wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -1,8 +1,21 @@
 # Nomination Assessment Team
 
-The **Nomination Assessment Team** (***NAT***) are the moderators of the [Beatmap Nominators](/wiki/People/Beatmap_Nominators). They are responsible for reviewing BN applications, creating tests for the applicants, evaluating current members and overseeing vetoes in the ranking process. All NAT members may act as BNs in the modding and ranking process, but they do not have strict activity requirements like BNs.
+The **Nomination Assessment Team** (**NAT**) are the moderators of the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (BN) and ensure the beatmapping side of osu! is functional.
```

Comment:
keep the italics on any first mentions of initialisms, I think it represents something like "we'll be referring to it as \_\_\_"